src/Makevars

```diff
@@ -1,3 +1,8 @@
 PKG_CXXFLAGS = -I. $(SHLIB_OPENMP_CXXFLAGS)
 PKG_LIBS = $(SHLIB_OPENMP_CXXFLAGS) $(LAPACK_LIBS) $(BLAS_LIBS) $(FLIBS)
 CXX_STD = CXX11
+
+# strip debug symbols for smaller Linux binaries
+strippedLib: $(SHLIB)
+		if test -e "/usr/bin/strip" & test -e "/bin/uname" & [[ `uname` == "Linux" ]] ; then /usr/bin/strip --strip-debug $(SHLIB); fi
```

Comment:
I have forgotten what the story was but I _think_ "they" decided to not allow this.  Unless you see it used in currently-published packages (RcppArmadillo does not, RcppEnsmallen does not, ...) I would recommend against.

I keep strip instruction in my `~/.R/Makevars` instead.

Reply:
I have removed "strip".